domains/joinGame/PendingGames.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { FlatList, TouchableOpacity, Text, View } from "react-native";
+import PropTypes from "prop-types";
+
+const PendingGames = ({ pendingGames }) => {
+    const renderPendingGame = ({ item }) => (
```

Comment:
Make it an external component. No need for it for have its own file, just move its declaration outside of the render method